doc/en/usage.rst

```diff
@@ -155,37 +155,14 @@ Example:
 .. code-block:: pytest
 
     $ pytest -ra
-    =========================== test session starts ============================
+    ================================ test session starts =================================
     platform linux -- Python 3.x.y, pytest-4.x.y, py-1.x.y, pluggy-0.x.y
+    hypothesis profile 'default' -> database=DirectoryBasedExampleDatabase('$REGENDOC_TMPDIR/.hypothesis/examples')
     rootdir: $REGENDOC_TMPDIR, inifile:
-    collected 7 items
```

Comment:
Weird, this is how it looks on `master` as well: https://docs.pytest.org/en/latest/usage.html

Reply:
Fixed, it seems this was not correct ever 😅